packages/cli/README.md

```diff
@@ -5783,5 +5783,13 @@ _See code: [src/commands/schema/update.ts](https://github.com/SmartThingsCommuni
 
 # Configuration and Logging
 
+## Enable Debug Logging
+
+Debug logging can be enabled via the `ST_DEBUG` environment variable. This will log at debug level to the console as well as the default log file.
+
+```console
+ST_DEBUG=true smartthings <command>
```

Comment:
I used `SMARTTHINGS_PROFILE` for the environment variable used to set the profile and `SMARTTHINGS_TOKEN` for one that can be used for tokens. We should probably stick to spelling out `SMARTTHINGS` or change those to match this format.

Reply:
I will switch to `SMARTTHINGS_DEBUG`.